src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -3168,50 +3172,56 @@ export class ZoneServer2016 extends EventEmitter {
     }
   }
 
-  customizeDTO(client: Client) {
-    const DTOArray: Array<any> = [];
+  customizeStaticDTOs() {
+    console.time("customizeStaticDTOs");
```

Comment:
```suggestion
```

Reply:
This only happens on server start so I think it's fine to have the time there, the more objects we add (ex. fences) will add to this time so it's good to see how long it takes